kubernetes/job.yaml

```diff
@@ -10,10 +10,10 @@ spec:
     spec:
       containers:
       - name: prowler
-        image: docker.io/prowler/kubernetes
+        image: toniblyx/prowler:latest
```

Comment:
Wouldn't be better to use `stable`? Latest is a development version.

Reply:
Sure, thanks!